src/views/playerdetail.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPlayersById } from '../services/fetchplayers';
+import './playerdetail.css';
+
+export default function PlayerDetail() {
+  const params = useParams();
+  const [player, setPlayerDetail] = useState({});
+  const [loading, setLoading] = useState(true);
+  //   console.log(params.id);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await getPlayersById(params.id);
+      setPlayerDetail(data);
+      setLoading(false);
+    };
+    fetchData();
+  }, [params.id]);
+
+  if (loading) return <div className="loader"> Loading Player</div>;
+  //   console.log(player);
+
+  return (
+    <div>
+      <p>
+        Click <a href="/player">Here</a> to return to the Players
```

Comment:
Be sure to replace all a elements with Link or NavLink when using react-router and linking routes.